_posts/2020-08-03-roadmap.md

```diff
@@ -0,0 +1,23 @@
+
+As we look forward to this next milestone, it is time to consider the next directions of MDAnalysis. The development of MDAnalysis has always been driven by the growing need for standardised, accessible analysis tools for open, reproducible, and collaborative research. While many major packages for molecular dynamics simulation provide their own set-up and analysis software, these are necessarily targeted to their own particular standards. MDAnalysis aims to provide analysis tools for simulation data in general, so historically a key objective has been to expand the number of supported package-specific data formats. As of version 1.0.0, we support 28 file formats used in major packages for both molecular dynamics and quantum chemistry. 
+
+In 1.0.0 we also began to explore an exciting new approach: direct interoperability with other popular packages for molecular analysis. [Converters](https://www.mdanalysis.org/docs//documentation_pages/converters.html) are distinct from [topology parsers](https://www.mdanalysis.org/docs//documentation_pages/topology_modules.html) and [coordinate readers](https://www.mdanalysis.org/docs//documentation_pages/coordinates_modules.html) as a third avenue for loading data into MDAnalysis. In 1.0.0 we added converters for two libraries: the molecular editor ParmEd, and chemfiles, a library for reading data from computational chemistry formats.
+
+The general lack of interoperability between software packages in the molecular modelling community has been highlighted in the [2019 report of the NSF MolSSI on Molecular Dynamics Software Interoperability](https://drive.google.com/file/d/1HKRhfm1Ev1UMAdvvuXhorSCUJlhtU-1o/view), noting consequences such as great duplication of effort in developing and maintaining similar tools across different formats; significant barriers to collaborating and transferring data; and requiring scientists to learn multiple packages and languages to access the full breadth of available analysis algorithms. 
+
+Moving forward, our plan is to increase the range of analyses and formats accessible to users by becoming interoperable with other relevant libraries. This reduces the need to duplicate and support existing tools within our own framework, and allows MDAnalysis to become a general-purpose analysis toolkit. We are already in the process of expanding compatible libraries in 2.0.0 by adding support for the widely popular [RDKit](https://www.rdkit.org/) cheminformatics toolkit via one of our [Google Summer of Code projects](https://www.mdanalysis.org/2020/05/20/gsoc-students/#c%C3%A9dric-bouysset-from-rdkit-to-the-universe-and-back) being carried out by Cédric Bouysset (@cbouy).
+
+Other targets for future releases include widely-used analysis libraries, such as [MDTraj](http://mdtraj.org/); libraries that can expand the range of formats we can support, such as [OpenBabel](http://openbabel.org/wiki/Main_Page); and direct interfacing with MD engines such as [OpenMM](http://openmm.org/).
```

Comment:
To make it a Roadmap, we need to add some timeline so I would add

> By the end of 2021, we plan to have added support for ...

This will also say "CZI, give us money, please".